tests/build/chains.go

```diff
@@ -157,6 +167,8 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", func() {
 
 				// Since specs could update the config policy, make sure it has a consistent
 				// baseline at the start of each spec.
+				var baselinePolicies = []string{"not_useful"}
+				g.GinkgoWriter.Printf("Set the non-blocking checks to baseline policies: %s\n", strings.Join(baselinePolicies, ", "))
 				Expect(kubeController.CreateOrUpdateConfigPolicy(
 					namespace, `{"non_blocking_checks":["not_useful"]}`)).To(Succeed())
```

Comment:
It would be great if the value we use in the call to `CreateOrUpdateConfigPolicy` and the value used in the call to `Printf` comes from the variable. Otherwise, the data is very likely to get out of sync in the future. Consider just creating a shared variable:
```
baselinePolicies := `{"non_blocking_checks":["not_useful"]}`
```